src/main/java/com/car/app/model/BaseModel.java

```diff
@@ -14,6 +14,8 @@
 
 import org.hibernate.annotations.GenericGenerator;
 
+import com.sun.istack.NotNull;
```

Comment:
I think you're using the wrong NotNull annotation. Please use the annotations from the package: javax.validation.constraints

Reply:
oh! yeah. I made a mistake in the import :D